### DISCUSSION TITLE
When will there be a new release?

### DISCUSSION THREAD

**Discussion by singhamitoj** [colobot/colobot#1384] [Q&A] [answered]
It is almost two years and no new Colobot release. Do you know when there will be a new version, and will it add new features? The update with the most content is the code battle update, so maybe you could add a new game mode, for example, multiplayer mode? Also, why is Colobot still in alpha development? I am thinking that we should be at least in the beta stages by now.

**Comment by Erihel** [comment]
There is a discord server where you can ask questions like this https://discord.gg/56Fm9kb Sadly this link is in dev branch and not in master branch yet.

There are mostly bug fixes in dev branch. As for alpha in the name I think it could be dropped.

**Comment by MrSimbax** [comment, marked as answer]
The best answer I can give you right now is: when it'll be ready then it'll be ready.

If you can't wait and want to play the most recent (although WIP) version of the game, then you can build yourself the [dev](https://github.com/colobot/colobot/tree/dev) branch (or any other branch, for that matter). In some cases you can also download a debug binary from [Actions](https://github.com/colobot/colobot/actions).

You can follow the progress of the next release or next feature set by looking at [projects](https://github.com/colobot/colobot/projects), [organization projects](https://github.com/orgs/colobot/projects/1) or [milestones](https://github.com/colobot/colobot/milestones). You can also take a look at the [commit history](https://github.com/colobot/colobot/commits/dev).

I'd also like to remind everyone that this is an open source project and there are only volunteers here, working in their spare time only whenever they want. There are a lot of feature requests (multiplayer is one of the most recurring ones) and we all would like for this game to have a lot of those features but those things take time and energy most people don't have an abundance of. Anyone can help by forking the project and submitting a [pull request](https://github.com/colobot/colobot/pulls).

Regarding the question about why we're still in the alpha stage, I'd say it's a matter of perspective. A lot of people want to add a lot of new features to the game, maybe even redesign some of them, so for them the game is still in the alpha stage. On the other hand, the game technically works and is complete (it was shipped in 2001 after all) so slapping on it the v1.0.0 or -beta tag seems appropriate too.
